src/Features/Core/Portable/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -294,6 +295,32 @@ private void AnalyzeInvocationOperation(OperationAnalysisContext operationContex
                 OnSymbolUsage(targetMethod, ValueUsageInfo.Read);
             }
 
+            private void AnalyzeNameOfOperation(OperationAnalysisContext operationContext)
+            {
+                // Workaround for https://github.com/dotnet/roslyn/issues/19965
+                // IOperation API does not expose potential references to methods/properties within
+                // a bound method group/property group.
+                var nameofArgument = ((INameOfOperation)operationContext.Operation).Argument;
+
+                if (nameofArgument is IMemberReferenceOperation)
+                {
+                    // Already analyzed in AnalyzeMemberReferenceOperation.
+                    return;
+                }
+
+                var symbolInfo = nameofArgument.SemanticModel.GetSymbolInfo(nameofArgument.Syntax, operationContext.CancellationToken);
+                foreach (var symbol in symbolInfo.GetAllSymbols())
+                {
+                    switch (symbol.Kind)
+                    {
+                        case SymbolKind.Method:
+                        case SymbolKind.Property:
+                            OnSymbolUsage(symbol, ValueUsageInfo.NameOnly);
```

Comment:
why only these types of symbols?

Reply:
Workaround is for missing IOperation for method groups and property groups.